### ISSUE TITLE
Is there any plan to publish a new release version in the near future？

### ISSUE THREAD

**Issue by mooooonlight** [microsoft/Detours#163]
The last release version data was  2018/4/17,  unfortunately if no new release version published in the near future, We'll not be able to use it anymore ><

**Comment by bgianfo** [comment]
I think we can cut a new release.
There are a few PRs I would like to get time to merge before we do so.

@dtarditi Do you have any thoughts here?

**Comment by dtarditi** [comment]
Yes, I agree we can cut a new release.  I also have a PR that I would like to merge before we do so.

**Comment by bgianfo** [comment]
I created a new github project to the [Next Major Release](https://github.com/microsoft/Detours/projects/1). 

@dtarditi can you add the PR you would like to merge to the board, so we can tracked them all?

**Comment by bgianfo** [comment]
To keep the thread updated are making progress on the release, we are finalizing the changes / PRs / issues to make sure we get everything we need.

**Comment by mooooonlight** [comment]
Thx very much! looking forward new release version !

**Comment by mooooonlight** [comment]
hello, Are there any developments or plans about next major release version? We're looking forward to a new version update.

**Comment by DrusTheAxe** [comment]
ping

Also see [Detours via Nuget package ? #124](https://github.com/microsoft/Detours/issues/222)

**Comment by sylveon** [comment]
@bgianfo any update on a new release?
